project-file/Shahzaib-Arshad-SP18-BSE-106.md

```diff
@@ -0,0 +1,28 @@
+# Project Name
+
+## Project Owner
+### Full Name
+Shahzaib Arshad
+
+### Roll Number
+SP18-BSE-106
+
+## Project Details
+### Description
+  The Project we are working on is EasyBus Portal. The purpose EasyBus Portal is to assist university transport management system to create routes and to manage the drivers, busses and student’s registration etc. Students can get live location of their bus, mark their attendance, check their ride and fee history. Moreover, students can submit suggestions and complain to the management through their portal. Parents will also be able to check bus  location, ride and fee history.
+
+### Technology Stack
```

Comment:
@shahzaib-insari Please mention all the technologies involved in an unordered list.